.github/workflows/prbuild.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
       - name: test
         run: |
           go mod vendor
-          make local-cluster install deploy test
+          make local-cluster test test-e2e
```

Comment:
in the future you might consider splitting these into separate jobs so they can run in parallel, but it's probably not a big deal right now.

Reply:
@skriss that's a great idea. I'd like to keep `prbuild.yaml` as-is for this PR and I'll create a follow-on PR to split the jobs.